bugis/description.md

```diff
@@ -0,0 +1,74 @@
+# Bugis
+
+## Introduction
+
+The Bugis are an ethnic group on the island of Sulawesi in Indonesia, with a strong seafaring and trading tradition. This sky culture has 12 constellation line figures. It also has names for four stars and five other sky objects (the Pleiades, the Coalsack Nebula, the Magellanic Cloud(s), the Milky Way, and Venus). Many of the constellations reflect the seafaring tradition; the others are of agricultural origin.
+
+## Description
+
+This sky culture for Stellarium is a transcript of data recorded in [#1]. This provides a unified drawing of Bugis constellations from two prior sources (one from 1999, one from 1987) which had overlapping but not identical data.
+
+The Mandar sky culture is related (they are neighbouring ethnic groups on the island of Sulawesi) so it is similar. The Mandar source [#2] also provides some data for the Bugis.
+
+### Constellations and star names
+
+One source attributed in [#1] (from 1999) gives the following Bugis asterisms and names:
+
+- _Bintoѐng Baluѐ_, `Widow Before Marriage`, (α, β Cen)
+- _Bintoѐng Bola Kѐppang_, `Incomplete House` (α to δ, μ Cru)
+- _Bintoѐng Balѐ Mangiwѐng_, `Shark` (Scorpius (south))
+- _Bintoѐng Lambaruѐ_, `Ray Fish` or `Skate` (Scorpius (north))
+- `Lost Pleiad` (Antares)
+- _Bintoѐng Kappala'ѐ_, `Ship` (α to η UMa)
+- _Bintoѐng Kappala'ѐ_, `Ship` (α to η UMa; β, γ UMi)
+- _Bintoѐng Balu Mandara'_, `Mandar Widow` (α, β UMa)
+- _Bintoéng Timoro'_, `Eastern Star` (Altair)
```

Comment:
I superficially wonder about the accent direction, Bintoéng or Bintoèng? Is this correct?

Reply:
Well caught! There are 2 sources, using different accents the same number of times each. It's not clear which is correct. 

The rising accent in "Bittoéng" (the same word in the related Mandar language) is more frequently documented in the same sources, and that's more likely.